lib/contractDeployer.js

```diff
@@ -0,0 +1,41 @@
+
+const fs = require('fs');
+const path = require('path');
+const { btcToWeis } = require('@rsksmart/btc-eth-unit-converter');
+const solUtils = require('./sol-utils');
+const TEST_RELEASE_BTC_CONTRACT = '../contracts/CallReleaseBtcContract.sol';
+const TEST_RELEASE_BTC_CONTRACT_NAME = 'CallReleaseBtcContract';
```

Comment:
If this a generic contract deployer perhaps these values should be received by param then?

Reply:
The `deployCallReleaseBtcContract` function is not generic. Look at the name. The idea for this file is that if we need to deploy more specific contracts in the future, we can create a specific function for that.
What do you think? I can make it generic, no problem. Let me know.

Reply:
Ok, thanks. I know there are contracts being deployed in other tests, perhaps we could eventually find a way to somehow concentrate that logic here. But out of the scope for now, I'll just create a ticket to analyze further